tubular/nominal.py

```diff
@@ -554,7 +560,11 @@ def __init__(
 
         BaseNominalTransformer.__init__(self, columns=columns, **kwargs)
 
-    def _prior_regularisation(self, target_means, cat_freq):
+    def _prior_regularisation(
+        self,
+        target_means: pd.Series,
+        cat_freq: str,
```

Comment:
I think this should be a pd.series, needs changing in the docstring as well.

Reply:
The object class name is capitalised - see [the docs](https://pandas.pydata.org/docs/reference/api/pandas.Series.html) - in keeping with [PEP8](https://peps.python.org/pep-0008/#class-names).

> Class names should normally use the CapWords convention.
> 
> The naming convention for functions may be used instead in cases where the interface is documented and used primarily as a callable.

No change needed?